src/view/QuestionSearchTag.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+
+const QuestionSearchTag = ({ questionsByTags  }) => (
```

Comment:
There is a lot of similarity between this view and the other question list. Think about extracting the common elements into reusable dumb components.